ibm/service/dnsservices/resource_ibm_private_dns_custom_resolver_secondary_zone.go

```diff
@@ -0,0 +1,269 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package dnsservices
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	pdnsSecondaryZoneID           = "secondary_zone_id"
+	pdnsSecondaryZoneZone         = "zone"
+	pdnsSecondaryZoneTransferFrom = "transfer_from"
+	pdnsSecondaryZoneEnabled      = "enabled"
+	pdnsSecondaryZoneDescription  = "description"
+	pdnsSecondaryZoneCreatedOn    = "created_on"
+	pdnsSecondaryZoneModifiedOn   = "modified_on"
+)
+
+func ResourceIBMPrivateDNSSecondaryZone() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMPrivateDNSSecondaryZoneCreate,
+		ReadContext:   resourceIBMPrivateDNSSecondaryZoneRead,
+		UpdateContext: resourceIBMPrivateDNSSecondaryZoneUpdate,
+		DeleteContext: resourceIBMPrivateDNSSecondaryZoneDelete,
+		Exists:        resourceIBMPrivateDNSSecondaryZoneExists,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
We don't have any wait logic added to wait for resource....You can remove the Timeouts block. We need them only if we have any wait functions where we need to wait for resource provision/updation/deletion